src/ExpressiveInstaller/Resources/src/App/Repository/PostRepository.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Post;
+use Kocal\Expressive\Database\Doctrine\DoctrineRepository;
```

Comment:
I used the namespace `Kocal\Expressive\...` because I was not sure if I was « allowed » to use `Zend\Expressive\...`